specification/source/glossary.rst

```diff
@@ -70,3 +85,13 @@ the Open Building Controls project.
    Mode
      In CDL, by mode we mean a signal that can take on multiple distinct
      values, such as ``On``, ``Off``, ``PreCool``.
+
+   Non-HVAC System
+     Any non-HVAC plant coupled with the control system.
+
+   Plant
```

Comment:
No, the definition relies on our OBC requirements document. In ISO 16484 the plant definition is broader and includes "a dedicated functional units for controls". However, the way the new terms are aligned should be internally consistent in the scope of OBC project.